nvdaHelper/local/nvdaHelperLocal.cpp

```diff
@@ -19,21 +19,25 @@ This license can be found at:
 #include <rpc.h>
 #include <sddl.h>
 #include <common/log.h>
+#include <common/apiHook.h>
 #include <local/nvdaControllerInternal.h>
 #include "nvdaHelperLocal.h"
 #include "dllImportTableHooks.h"
 #include "rpcsrv.h"
 
-DllImportTableHooks* oleaccHooks = NULL;
-DllImportTableHooks* uiaCoreHooks = NULL;
+decltype(&SendMessageW) real_SendMessageW = nullptr;
+decltype(&SendMessageTimeoutW) real_SendMessageTimeoutW = nullptr;
+decltype(&OpenClipboard) real_OpenClipboard = nullptr;
+
+bool isSecureModeNVDAProcess=false;

```

Comment:
nit: Do we really need this global? I don't think anything uses it outside of nvdaHelperLocal_initialize.

Reply:
Indeed, we might as well remove it. I copied it from remote and didn't realise that explicit deinit of hooks is not necessary.